### ISSUE TITLE
Error using `eaf-open`

### ISSUE THREAD

**Issue by Compro-Prasad** [manateelazycat/emacs-application-framework#5]
From buffer `*eaf*`
```
Traceback (most recent call last):
  File "/home/compro/.spacemacs.d/emacs-application-framework/core/eaf.py", line 174, in <module>
    bus = dbus.SessionBus()
  File "/usr/lib/python3.6/site-packages/dbus/_dbus.py", line 211, in __new__
    mainloop=mainloop)
  File "/usr/lib/python3.6/site-packages/dbus/_dbus.py", line 100, in __new__
    bus = BusConnection.__new__(subclass, bus_type, mainloop=mainloop)
  File "/usr/lib/python3.6/site-packages/dbus/bus.py", line 122, in __new__
    bus = cls._new_for_bus(address_or_type, mainloop=mainloop)
dbus.exceptions.DBusException: org.freedesktop.DBus.Error.BadAddress: '=' character not found or has no value following it
```
Using it on Arch Linux, GNOME.

**Comment by Compro-Prasad** [comment]
Running it from within emacs doesn't work.

**Comment by manateelazycat** [comment]
Did you download subdirectory "app" and "core" in same directory of eaf.el ?

**Comment by Compro-Prasad** [comment]
Yep.

**Comment by Compro-Prasad** [comment]
Running from Emacs surely doesn't work. I have tried many ways like eshell, async-shell-command, shell-command, term, shell.

**Comment by manateelazycat** [comment]
How do you use emacs? Use emacs in terminal or as x11 window?

**Comment by Compro-Prasad** [comment]
Emacs(spacemacs) as an application in GNOME.

**Comment by Compro-Prasad** [comment]
I can't get this too. Debugging in Emacs I have to use `obj.__dir__(1)` while on command line I have to do `obj.__dir__()`. Vice versa throws an error.

**Comment by manateelazycat** [comment]
Can you run "emacs -Q" to test agian? 
Perhpas spacemacs has something make eaf.el going wrong

**Comment by manateelazycat** [comment]
Looks it's not bug of eaf.el or eaf.py.

It's bug of spacemacs, spacemacs is too big, you can uncomment it's code to find problem make dbus can't work.

I need spend my time on develop, haven't time do that, sorry.

**Comment by AlessandroW** [comment]
I had the same issue when using spacemacs, my best guess is that you have to load the file the spacemacs way.
See: http://spacemacs.org/doc/DOCUMENTATION.html#without-a-layer

**Comment by csfreebird** [comment]
> Yep. It works. Can you help me find what is causing this?

I ran into the problem today, in Linux terminal, launch pyqt application--a simple browser program, it works. but in spacemacs, it crashes. 

I have used proxychains4 to launch spacemacs, and spacemacs.env save some env variables for proxychains. So my spacemacs automatically use proxychains4, this cause my pyqt program to crash.
It works after deleting spacemacs.env file.
